release/tigervnc.iss.in

```diff
@@ -24,10 +24,40 @@ LicenseFile=@CMAKE_SOURCE_DIR@\LICENCE.txt
 Name: "{sys}\config\systemprofile\Desktop"
 
 [Files]
+#ifndef BUILD_STATIC
+Source: "@WIN_DLL_PATH@libintl-8.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libiconv-2.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@mgwfltknox-1.3.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@zlib1.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libjpeg-8.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libgcc_s_seh-1.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libstdc++-6.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libwinpthread-1.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+#ifdef ENABLE_GNUTLS
+Source: "@WIN_DLL_PATH@libffi-7.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libgmp-10.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libgnutls-30.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libhogweed-6.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;  
+Source: "@WIN_DLL_PATH@libidn2-0.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libnettle-8.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libp11-kit-0.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libtasn1-6.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libunistring-2.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+#endif
+#else
+Source: "@WIN_DLL_PATH@libffi-7.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libp11-kit-0.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libunistring-2.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+Source: "@WIN_DLL_PATH@libintl-8.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+Source: "@WIN_DLL_PATH@libiconv-2.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace;
+#endif
 #ifdef BUILD_WINVNC
 Source: "@CMAKE_CURRENT_BINARY_DIR@\win\winvnc\winvnc4.exe"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
 Source: "@CMAKE_CURRENT_BINARY_DIR@\win\wm_hooks\wm_hooks.dll"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
 Source: "@CMAKE_CURRENT_BINARY_DIR@\win\vncconfig\vncconfig.exe"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+#ifndef BUILD_STATIC
+Source: "@CMAKE_CURRENT_BINARY_DIR@\win\vncconfig\vncconfig.exe"; DestDir: "{app}"; Flags: ignoreversion restartreplace; 
+#endif
```

Comment:
Isn't this just a duplicate of the line above?

Reply:
Oops.  Thanks for catching that.  I don't have a lot of experience in pull requests.  Do I just amend my previous commit to fix it?

Reply:
Yup. And then you need to do a force push as you've altered the history.

Reply:
OK.  I believe this item is fixed now.